lib/zendesk_apps_support/validations/svg.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+require 'loofah'
+
+module ZendeskAppsSupport
+  module Validations
+    module Svg
+      class << self
+        def call(package)
+          errors = []
+
+          strip_declaration = Loofah::Scrubber.new do |node|
+            node.remove if node.name == 'xml' && node.children.empty?
+          end
+
+          empty_malformed_markup = Loofah::Scrubber.new do |node|
+            node.next.remove while node.name == 'svg' && node.next
+          end
+
+          # CRUFT: ignore a (very specific) style attribute which Loofah would otherwise scrub.
+          # This attribute is deprecated (https://www.w3.org/TR/filter-effects/#AccessBackgroundImage)
+          # but is included in many of the test apps used in fixtures for tests in ZAM, ZAT etc.
+          remove_enable_background = Loofah::Scrubber.new do |node|
```

Comment:
all loofah scrubbers can be defined outside of `call` method?

Reply:
👍